<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Заказ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Изделие</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ЛЗ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Номер ЛЗ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество листов</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="update">
     <property name="text">
      <string>Обновить данные</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
